cypress/integration/test/create-user/create-user-tests.spec.js

```diff
@@ -0,0 +1,152 @@
+/// <reference types="Cypress" />
+
+import {adaptToReduxPersist} from '../../utils/redux'
+import DashboardPage from '../../page/dashboard-page'
+import UserPage from '../../page/user-page'
+import Utils from '../../utils/utils'
+import createUserCore from '../helpers/create-user/user-core'
+
+const utils = new Utils()
+const userPage = new UserPage()
+const dashboardPage = new DashboardPage()
+
+let userData
```

Comment:
Since `userData` and `loginCredentials` are used inside tests in here 

```
describe('create users suite', function() {
....
}
```

then its better to move them inside the describe function block, so that it can be auto cleaned up